### ISSUE TITLE
delete record

### ISSUE THREAD

**Issue by charlieridley** [charlieridley/emu#16]

**Comment by hathvi** [comment]
Hey @charlieridley , was just wondering if you had any plans of adding a delete method to the `Emu.Model` and not just the `Emu.ModelCollection`. Right now I'll likely end up just using the stores `deleteRecord` and maybe just extending `Emu.Model` and adding some of these methods until they're implemented. I'm crazy busy right now or else I'd create some pull requests for you. I'm not a huge fan of the CoffeeScript though, but hopefully here in a month or so I'll have some more time to actually contribute some.

P.S. Could you maybe create a reference to which commits fix / close issues or just comment on why the issue was closed? Thanks for this great project!

**Comment by charlieridley** [comment]
Yes, I could proxy the delete method to the store, will reopen this.
